pybotvac/robot.py

```diff
@@ -234,6 +235,10 @@ def __init__(self, serial, secret):
         self.secret = secret
 
     def __call__(self, request):
+        # Due to https://github.com/stianaske/pybotvac/issues/30
+        # Neato expects and supports authentication header ONLY for en_US
+        locale.setlocale(locale.LC_ALL, 'en_US.utf8')
```

Comment:
this is the real fix;  obviously forcing locale is unfortunate ... but if the vendor (Neato) doesn't support any other locale other than `en_US`, I don't think there is anything else we can do

(perhaps the RIGHT thing would be to submit a bug report to them)

Reply:
one consideration is that we might want to UNSET the locale, let me know if that is desired

Reply:
Hey @fermulator. Thank you for this fix. Unfortunately this causes errors on systems, where en_US.UTF-8 is not available. Can you use pytz instead? https://pypi.org/project/pytz/